website/util/sanitize.py

```diff
@@ -74,6 +74,18 @@ def safe_unescape_html(s):
         '&lt;': '<',
         '&gt;': '>',
     }
-    for escape_sequence, character in safe_characters.items():
-        s = s.replace(escape_sequence, character)
+    if isinstance(s, dict):
+        return {
+            key: safe_unescape_html(value)
+            for (key, value) in s.iteritems()
+        }
+    if isinstance(s, list):
```

Comment:
This should be able to handle other collection types besides lists, e.g. tuples. You could pass a tuple to `isinstance`

```python
if isinstance(s, (list, tuple)):
    #...
```

Though it is generally more Pythonic to use duck-typing rather than `isinstance`. For example, you could use a helper like:


```python
def is_iterable_but_not_string(obj):
    return (hasattr(obj, '__iter__') and not hasattr(obj, 'strip'))
```